tests/components/roborock/snapshots/test_vacuum.ambr

```diff
@@ -0,0 +1,24 @@
+# serializer version: 1
+# name: test_get_maps
+  dict({
+    'vacuum.roborock_s7_maxv': dict({
+      'maps': list([
+        dict({
+          'flag': 0,
+          'name': 'Upstairs',
+          'rooms': dict({
+            16: 'Example room 1',
+            17: 'Example room 2',
+            18: 'Example room 3',
+          }),
+        }),
+        dict({
```

Comment:
I suppose this is empty as it's not the current active map?

Reply:
So this was actually a side effect of the map flag not being updated in the coordinator. It gets updated on status change - but a) that doesn't happen in tests as status is static and b) in the real system that also wouldn't always happen before setting the room info. so I manually set the current map now.